# -*- coding: utf-8 -*-
# Generated by Django 1.9.4 on 2017-12-12 07:24
from __future__ import unicode_literals

import datetime
from django.db import migrations, models
from django.utils.timezone import utc


class Migration(migrations.Migration):

    dependencies = [
        ('dreamcode', '0005_auto_20171212_0619'),
    ]

    operations = [
        migrations.AlterField(
            model_name='contest',
            name='end',
            field=models.DateTimeField(default=datetime.datetime(2017, 12, 12, 7, 24, 40, 711625, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='contest',
            name='start',
            field=models.DateTimeField(default=datetime.datetime(2017, 12, 12, 7, 24, 40, 711571, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='testresult',
            name='result',
            field=models.CharField(choices=[('PD', 'Pending'), ('OK', 'Accepted'), ('PE', 'Presentation error')], default='PD', max_length=2),
        ),
        migrations.AlterField(
            model_name='testresult',
            name='status',
            field=models.CharField(choices=[('PD', 'Pending'), ('OK', 'Okay'), ('RF', 'Restricted Function'), ('ML', 'Memory Limit Exceed'), ('OL', 'Output Limit Exceed'), ('TL', 'Time Limit Exceed'), ('RT', 'Run Time Error (SIGSEGV, SIGFPE, ...)'), ('AT', 'Abnormal Termination'), ('IE', 'Internal Error (of sandbox executor)'), ('BP', 'Bad Policy'), ('R0', 'Reserved result type 0'), ('R1', 'Reserved result type 1'), ('R2', 'Reserved result type 2'), ('R3', 'Reserved result type 3'), ('R4', 'Reserved result type 4'), ('R5', 'Reserved result type 5'), ('EX', 'Exception'), ('FA', 'Failed')], default='PD', max_length=2),
        ),
    ]
